packages/helper-grammar-regex-collection/index.js

```diff
@@ -2,6 +2,8 @@ import regex from './regex';
 
 export { default as go } from './go.js';
 
+const test1 = 'a';
```

Comment:
<!--
  1 failure:  'test1' is assign...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: packages/helper-grammar-regex-collection/index.js;
  Line: 5;
-->  
- :no_entry_sign: 'test1' is assigned a value but never used.